tests/test_forms.py

```diff
@@ -1,14 +1,14 @@
-from datetime import datetime
+from datetime import datetime, timezone
 
 from django import forms
 from django.test import TestCase
 
 from debug_toolbar.forms import SignedDataForm
 
-SIGNATURE = "v02QBcJplEET6QXHNWejnRcmSENWlw6_RjxLTR7QG9g"
+SIGNATURE = "-WiogJKyy4E8Om00CrFSy0T6XHObwBa6Zb46u-vmeYE"
 
-DATA = {"value": "foo", "date": datetime(2020, 1, 1)}
-SIGNED_DATA = f'{{"date": "2020-01-01 00:00:00", "value": "foo"}}:{SIGNATURE}'
+DATA = {"value": "foo", "date": datetime(2020, 1, 1, tzinfo=timezone.utc)}
```

Comment:
Again, this changes the naive datetime to aware, so that it isn't interpreted with the default timezone when converting to str